hack/release/common.sh

```diff
@@ -110,6 +104,41 @@ publishHelmChart() {
   cosignOciArtifact "${version}" "${commit_sha}" "${build_date}" "${oci_repo}${helm_chart}:${version}@${helm_chart_digest}"
 }
 
+updateAhConfig() {
+  local oci_repo helm_chart ah_config_path image_config_path image_config media_type oci_repository oci_image old_config_digest blob_digest
+
+  oci_repo="${1}"
+  helm_chart="${2}"
+
+  ah_config_path="./charts/${helm_chart}/artifacthub-repo.yml"
```

Comment:
```suggestion
  ah_config_path="./charts/${helm_chart}/artifacthub-repo.yaml"
```

This name doesn't match the current filename

Reply:
@jonathan-innis this should be resolved now.